htdocs/core/lib/admin.lib.php

```diff
@@ -1588,12 +1588,12 @@ function form_constantes($tableau, $strictw3c = 0, $helptext = '')
 	                		$moreonlabel = '';
 	                		if (!empty($arrayofmessagename[$modelmail->label])) $moreonlabel = ' <span class="opacitymedium">('.$langs->trans("SeveralLangugeVariatFound").')</span>';
 	                		// The 'label' is the key that is unique if we exclude the language
-	                		$arrayofmessagename[$modelmail->label.':'.$tmp[1]] = $langs->trans(preg_replace('/\(|\)/', '', $modelmail->label)).$moreonlabel;
+	                		$arrayofmessagename[$modelmail->label] = $langs->trans(preg_replace('/\(|\)/', '', $modelmail->label)).$moreonlabel;
```

Comment:
I suppose the :xxx is used by other setup page that need to now the typeof email template selected not only its label.
A more secured fix would be to clean the :member part if we don'tneed it into the constant when we record the constant. Suggested change seems to create regression into other features.